virus/virus.pde

```diff
@@ -405,6 +405,7 @@ void drawUI(){
   if(selectedCell != null){
     drawCellStats();
   }
+  //image(loadImage("nygi6iodek151.jpg"),0,0);

```

Comment:
Removing the two changes to this file would be nice

Reply:
agreed

Reply:
shit how do i cancel that